app/views/TaxYearSummary.scala.html

```diff
@@ -266,14 +306,16 @@ <h1 class="govuk-heading-xl" id="heading">
         @messages("tax-year-summary.heading")
     </h1>
     @modelOpt.map { model =>
-        @if(!model.crystallised.contains(true)) {
+        @if(!model.crystallised.contains(true) && (((!isAgent) && model.messages.isEmpty|| model.messages.exists(_.errorMessages.size == 0)))) {
```

Comment:
It might read better like this:
`val errorsPresent = model.messages.exists(_.....
@if(!model.crystallised.contains(true) && !errorsPresent)`

Reply:
you might have to pull this out to a function though..